102.md

```diff
@@ -0,0 +1,107 @@
+NIP-102
+=======
+
+Subkey Attestation and Management
+------------------------------
+
+`draft` `optional`
+
+This NIP defines a way to separate identity from authentication using subkey attestations. This allows the use of one keypair to issue independent keypairs for different apps. If any subkey key is compromised, the root keypair can publish an event revoking the key.
+
+## Attestation
+
+A subkey attestation is created by signing the message:
+
+```
+nip102:<hex-subkey1-pubkey>
+```
+
+This signature is then added to events to demonstrate authorization to post on behalf of the original key.
+
+The parent key can publish a `Kind 10102` event for subkey management. This event lists attestations that have been revoked, as well as those that should be used to encrypt messages so that they can be read using preferred clients. These declarations are only a preference, as they may not be available in a timely manner.
+
+Content:
+```json
+{
+  "inbox_keys": ["<hex-subkey1-pubkey>", "<hex-subkey2-pubkey>"],
+  "revoked_subkeys": ["<hex-subkey3-pubkey>"]
+}
+```
+
+## Event Signing
+
+When signing events using a subkey, the account pubkey will be included as a well-known attribute. The event will also include an attestation of the subkey's authorization signed by the main key. Relays and clients should validate this attestation, and discard messages that make invalid claims.
+
+```json
+{
+  "pubkey": "<hex-subkey1-pubkey>",
+  "kind": 1,
+  "tags": [
+    ["I", "<hex-account0-pubkey>"],
+    ["Ia", "<subkey1-attestation>"]
```

Comment:
I think this is correct because we won't search for the attestation. If there are other reasons to use single characters I was considering `J`.